### DISCUSSION TITLE
How to Eexport animation loop for web?

### DISCUSSION THREAD

**Discussion by liquidia** [senshu/Sozi#609] [Q&A]
Hello All,

I am new to sozi, just wanted to know if there is any option to export as svg animation for a landing/ web page? thats starts onload and not like presentation.

**Comment by tshoppa** [comment]
Hi,
Sozi does not manipulate the svg but rather creates a html file with the embedded presentation. If you want to insert it into your own html page, you can add it with an iframe.
Running a slideshow automatically is quite simple, just add a timeout to each frame:

- Activate the timeout by clicking the clock icon next to "timeout"
- Set a proper time the frame should be displayed before transitioning to the next frame.

![image](https://user-images.githubusercontent.com/75733658/164412691-39974a66-e7e0-439a-99f7-b416bce721ec.png)

**Comment by aumouvantsillage** [comment]
As a complement to @tshoppa's answer, I think that the following documents can help:

* [Embedding Sozi presentations in HTML documents](https://sozi.baierouge.fr/pages/tutorial-embedding.html).
* [The "first presentation" tutorial](http://sozi.guide/en/first-presentation.html) in the Sozi Guide. See section [Take control of space and time](http://sozi.guide/en/first-presentation.html#sec-5) for explanations on automatic transitions.
